spec/features/bulk_updates/bulk_updates_instruments_spec.rb

```diff
@@ -197,8 +197,8 @@
         end
 
         it 'displays instrument changes' do
-          within '.collection-overview-table' do
-            expect(page).to have_no_content('ADS')
+          within '#metadata-preview' do
+            expect(page).to have_content('ADS')
```

Comment:
This is a "have_no_content" expect, so it should mimic it.  We might need to look at this one a little closer to figure out what to match on.

Reply:
Yeah this one is tricky because the test fails because there is 'ADS' in the content.

Reply:
So what do you recommend?